src/wallet/coinselection.h

```diff
@@ -36,6 +36,8 @@ class CInputCoin {
     COutPoint outpoint;
     CTxOut txout;
     CAmount effective_value;
+    CAmount m_fee;
```

Comment:
This commit makes `effective_value` a bit ambiguous to the reader now that both short and long-range fees are being stored.

Reply:
It makes everything easier to precompute and store these since they are used  in multiple places.

Reply:
I understand the motivation, my nitting is about naming since we're expanding the "scope" of what is being cached.